SIS_import/rclone_setup.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash -x 
 
+if [ -z "${aws_s3_bucket}" ]; then
+    echo "set to default"
```

Comment:
Minor: But I'd have this message say something like

`aws_s3_bucket set to default value of umich-tl-sis`

Reply:
updated. Please see the latest code